atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunnerImpl.java

```diff
@@ -73,6 +75,7 @@
  */
 public class SweepTaskRunnerImpl implements SweepTaskRunner {
     private static final Logger log = LoggerFactory.getLogger(SweepTaskRunnerImpl.class);
+    private static final int MAX_VALUES_TO_DELETE_BATCH = 10_000;
```

Comment:
I don't think this is needed.  We are using the broken jdbc KV store which is known to have these issues that other KV stores don't have